Eric-Obeng.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://github.com/EddieHubCommunity/awesome-github-profiles/assets/61484602/c870b265-48ff-414d-aabe-45b004f21326",
+    "issueId": "1870",
+    "name": "Eric Obeng",
+    "username": "Eric-Obeng"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

Looks like this is still missing. Do you need any help with this? 🙂

---

Please what actually is missing